test/test_onnx.py

```diff
@@ -92,6 +92,18 @@ def forward(self, boxes, scores):
 
         self.run_model(Module(), [(boxes, scores)])
 
+    def test_batched_nms(self):
+        class Module(torch.nn.Module):
+            def forward(self, boxes, scores, idxs):
+                return ops.batched_nms(boxes, scores, idxs, 0.5)
+
+        for num_boxes in (5, 100, 5_000):
+            boxes = torch.rand(num_boxes, 4)
+            boxes[:, 2:] += torch.rand(num_boxes, 2)
```

Comment:
```suggestion
            boxes[:, 2:] += boxes[:, :2]
```

Looks like the root cause for mismatch is that for many cases, the random box is not having x2 > x1 && y2 > y1. This leads to undefined behavior, and thus mismatch. I did a quick check with above change, and results are all matching. (although the above change is not entirely legal as well since it could potentially have x2,y2 > 1, but that help prove the assumption)

Reply:
Thanks @BowenBao , you are correct, the assumptions of `nms` for the dimensions weren't respected in the test. In fact `test_nms` was incorrect too so I fixed it as well.

>  it could potentially have x2,y2 > 1

I think that's fine, there's no upper limit on x2 and y2 - unless I'm missing something here?

Reply:
Thanks for your fix. Yes after checking the doc again i think it's fine. I thought for a moment the coordinates are required to be normalized to be within [0,1].